packages/vue-app/template/client.js

```diff
@@ -17,7 +17,7 @@ import {
   globalHandleError
 } from './utils.js'
 import { createApp, NuxtError } from './index.js'
-import NuxtLink from './components/nuxt-link.<%= router.prefetchLinks ? "client" : "server" %>.js' // should be included after ./index.js
+import NuxtLink from './components/nuxt-link.client.js' // should be included after ./index.js
```

Comment:
I don't completely understand what would be these three states:

- `prefetchLinks: true` (default) -> import `nuxt-link.client.js`
- `prefetchLinks: false` -> import `nuxt-link.server.js` (`prefetch` is completely disabled)
- `prefetchLinks: undefined` -> import `nuxt-link.client.js` (`:prefetch="true"` would work) 

Is that what you mean?

The code would be like that:

```js
import NuxtLink from './components/nuxt-link.<%= router.prefetchLinks === false ? "server" : "client" %>.js'
```

But I think it's really unclear 😕 because __the user would have to set `prefetchLinks: undefined` explicitly__. Your app.feature PR would really help to solve this 🙂.

Reply:
In your example I would probably switch the meaning of `false` and `undefined`. But still not optimal indeed, not sure what the best course of action is here. Lets wait for someone else to chime in :)

Reply:
I'd also say that switching the behavior and removing the prefetch part completely when set to `undefined` (or maybe even `null` to make it more intentional?) would make sense.

However, we could also work with string here.

Reply:
@pimlie I just rebased my branch to include the latest changes from `dev`. I think the https://github.com/nuxt/nuxt.js/pull/6287 solves this conversation. To disable prefetch completely and import `nuxt-link.server.js`, users will have to set `features.clientPrefetch` to false.